### DISCUSSION TITLE
When I make a storage deal is that content available on IPFS?

### DISCUSSION THREAD

**Discussion by Erotemic** [filecoin-project/lotus#8500] [General Q&A] [answered]
I'm looking for a way to archive about 17+GB of data while making it publicly available on IPFS.

I've spent the day learning how to use lotus and make filecoin deals, and I would have tested this myself, but my 512MB test case got rejected by the miners I attempted to make a deal with. It seems that I need to upload more data to get a successful deal.

But before I add the entire dataset, I wanted to know if making a deal for a miner to host data would also make that data available on IPFS (or if there was a way to make that happen).

For a specific use case, say:

* I have an IPFS node hosting CID: `QmfStoay5rjeHMEDiyuGsreXNHsyiS5kVaexSM2fov216j`, and nobody else is pinning this content. 
* I make a filecoin deal via lotus to have a miner store `QmfStoay5rjeHMEDiyuGsreXNHsyiS5kVaexSM2fov216j`
* I take my IPFS node offline.

Will the data still be available on ipfs.io/ipfs/QmfStoay5rjeHMEDiyuGsreXNHsyiS5kVaexSM2fov216j via the miner? Or is the only way to access that file again via requesting that CID from the miner explicitly via a filecoin implementation like lotus?

I'm expecting the answer to be no, because filecoin and IPFS are separate entities, but I figure I would ask the question, and if the answer is no, are there any existing ways to effectively pay someone in filecoin to host the data on IPFS. I'm currently using pinata, but that's a bit too centralized (and pricey) for my taste. I don't need blazing fast access to the data, but I do want to ensure that it is accessible beyond my self-hosted IPFS node.

**Comment by stuberman** [comment, marked as answer]
See https://pl-strflt.notion.site/Project-Thunder-Proposal-056fec9d4c5b4abfa34280b627d9e7dd

**Reply by Erotemic** [reply]
So the takeaway is that this feature is not available yet, but project thunder is a proposal for it that might be implemented as soon as September 2022?

**Comment by LexLuthr** [comment]
@Erotemic You can also look into https://web3.storage/ as a possible solution.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
